project3/styles.css

```diff
@@ -0,0 +1,321 @@
+    text-align: center;
+    color: #1e2019;
+
+}
+
+span {
+    display: block;
+}
+
+.shield {
+    width: 240px;
+    height: 239px;
+    display: block;
+    position: relative;
+    top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #cececd;
+}
+
+.icons {
+    padding: 10px;
+    text-align: center;
+}
+
+@media (min-width:900px) {
+    .icons {
+        display: inline-block;
+        width: 30%;
+    }
+
+    select {
+        cursor: pointer;
+        width: 80%;
+        height: 70px;
+        font-family: "Benchnine", monospace;
+        font-size: 18px;
+        text-align: center;
+        color: #1e2019;
+
+    }
+}
+
+@media (min-width:400px) {
+
+    select {
+        cursor: pointer;
+        width: 80%;
+        height: 70px;
+        font-family: "Benchnine", monospace;
+        font-size: 18px;
+        text-align: center;
+        color: #1e2019;
+
+    }
+}
+
+.bulb {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.clock {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.iconsubs {
+    padding-top: 25px;
+}
+
+.muscle {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+textarea {
+    height: 120%;
+    width: 75%;
+}
+
+textarea:hover {
+    border: 1px solid #71f79f;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-animation: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 1px solid #cececd;
+    height: 20px;
+    width: 20px;
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+
+input[type="checkbox"]:checked {
+    background-color: #71f79f;
+    border-color: #d1d5de;
+    font-size: #cececd;
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+}
+
+input[type="radio"]:checked {
+    background-color: #71f79f;
+    border-color: #d1d5de;
+    font-size: #cececd;
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+}
+
+input[type="checkbox"]:checked+label,
+input[type="radio"]:checked+label {
+    text-decoration: underline;
+    text-decoration-color: #71f79f;
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    background-color: #1e2019;
+    border: 1px solid #71f79f;
+}
+
+input[type="text"],
+input[type="email"],
+input[type="number"] {
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    color: #1e2019;
+    display: block;
+    width: 50%;
+}
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="number"]:focus {
+    color: #1e2019;
+}
+
+input[type="text"]:required:valid,
+input[type="email"]:required:valid,
+input[type="number"]:required:valid {
+    border-bottom-color: #71f79f;
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    color: #1e2019;
```

Comment:
You can delete these lines; you only need to write CSS for what is changing with state changes, not repeat existing styles.